### ISSUE TITLE
Create Adjustment page issues

### ISSUE THREAD

**Issue by Vertex21** [uilibrary/Stocky-Issues-and-Feature-request#335]
Hi! I have one suggestion and one issue with the Create Adjustment page. 

**The issue** is that the system will not allow you to add a product that has a zero quantity. I believe this page is for adjusting inventory but it is useless if I cannot adjust quantity zero items. Maybe it is not often that you will have a zero quantity item and then find that item on your shelf, but sometimes it does happen.

**The suggestion** I have is to change this Create Adjustment template, or create a new Inventory Adjustment template. When a person is taking inventory, he counts all of his items. On your Create Adjustment page it does not have fields to enter the quantity counted, only the quantity to adjust up or down. This means the person has to create the calculation for every item which introduces a lot of chances for error. It would be better to have a field to input the actual inventory stock quantity and then the system does the calculations for you.

Thank you!

**Comment by hamzakifani** [comment]
Hi. To add adjustment you have to purchase the product first. Because the adjustments comes when you have stock.

Adjustment can only occur when you have stock. Without stock how can you adjust quantity? So purchase that product first. Then you can make adjustment. Thank you

create purchase via http://stocky.ui-lib.com/app/purchases/store

Thanks

**Comment by Vertex21** [comment]
Yes, but consider this scenario. I purchase 10 items. My employee sells 10 items but 2 items are accidently left behind. In a few days we do inventory and find that we have the 2 items. How do I now create an adjustment to put the items back in inventory? I do not want to do a purchase for them as they aren't a purchase.
It would be easier to create an adjustment from zero to what I have in stock (2).

**Comment by hamzakifani** [comment]
you need to know why 2 items are accidently . if we adjust a product with stock 0 it no logic here and the GOGS profit calculator will be wrong . Thanks
